.github/workflows/linux-build-base.yml

```
@@ -18,45 +18,61 @@ on:
   workflow_call:
     inputs:
       use-clang:
-        description: 'Use Clang to compile the project.'
+        description: Use Clang to compile the project.
         default: false
         required: false
         type: boolean
 
 jobs:
   ubuntu-release:
-    name: "Ubuntu release with resolve_dependency"
+    name: Ubuntu release with resolve_dependency
     runs-on: yscope-gh-runner
     env:
-      CCACHE_DIR: "${{ github.workspace }}/ccache"
-      USE_CLANG: "${{ inputs.use-clang && 'true' || 'false' }}"
+      CCACHE_CONFIGPATH: ${{ github.workspace }}/ccache.conf
+      CCACHE_DIR: ${{ github.workspace }}/ccache
```

We can configure ccache using environment variables, to avoid needing to set up the runner with a config file. I'll explain in the other comment why we don't need to set CCACHE_DIR.

```suggestion
      CCACHE_COMPRESSLEVEL: 2
      CCACHE_MAX_SIZE: 5G
```